helpers/testdb/main.go

```diff
@@ -24,7 +25,7 @@ type Database struct {
 func GetTestDB() *Database {
 	if database == nil {
 		err := godotenv.Load()
-		if err != nil {
+		if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
`const` ?

Reply:
there's any sense?

Reply:
it's used just to ignore message `.env no such file or directory`